lib/internal/errors.js

```diff
@@ -204,6 +206,15 @@ const addCodeToName = hideStackFrames(function addCodeToName(err, name, code) {
   }
 });
 
+function isErrorStackTraceLimitWritable() {
+  const desc = ObjectGetOwnPropertyDescriptor(Error, 'stackTraceLimit');
+  if (desc === undefined) {
+    return ObjectIsExtensible(Error);
+  }
+
+  return desc.writable ?? typeof desc.set === 'function';
```

Comment:
We could try to cache the value when the property is not configurable:
```suggestion
  const returnValue = desc.writable ?? typeof desc.set === 'function';

  if (desc.configurable === false)
    module.exports.isErrorStackTraceLimitWritable = () => returnValue;

  return returnValue;
```

I think it makes the harder to read, and I think we should try to optimize for the most common case (which is when the property has `configurable` and `writable` set to `true`).

Reply:
I'm fine not cacheing for now